src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala

```diff
@@ -63,8 +58,57 @@ class KinesisImportQueryGeneratorIT
     assertResultSet(expected)
   }
 
+  private[this] def assertResultSet(expected: List[(Int, Int, String, String, Boolean)]): Unit = {
+    val resultSet = statement.executeQuery(s"SELECT * FROM $TEST_TABLE_NAME")
+    val values = collectResultSet(resultSet)(extractTuple)
+    assert(values === expected)
+    assert(resultSet.next() === false)
+    ()
+  }
+
+  private[this] def extractTuple(resultSet: ResultSet): (Int, Int, String, String, Boolean) =
+    (
+      resultSet.getInt("sensorId"),
+      resultSet.getInt("currentTemperature"),
+      resultSet.getString("status"),
+      resultSet.getString(KINESIS_SHARD_ID_COLUMN_NAME),
+      resultSet.getString(SHARD_SEQUENCE_NUMBER_COLUMN_NAME) != null
+    )
+
+  private[this] def putRecordIntoStream(
+    sensorId: Int,
+    currentTemperature: Int,
+    status: String,
+    partitionKey: String,
+    streamName: String
+  ): Unit = {
+    val recordData =
+      s"""{\"sensorId\": $sensorId,
+         | \"currentTemperature\": $currentTemperature,
+         | \"status\": \"$status\"
+         | }""".stripMargin.replace("\n", "")
```

Comment:
You should be able to remove the escapes inside string interpolation. Could you please check it and remove `\"`?

Reply:
it worked!

Reply:
Great :+1: